distribution/docs/src/main/resources/content/_managing/attributes-added-by-input-transformers.adoc

```diff
@@ -0,0 +1,26 @@
+:title: Attributes Added by Input Transformers
+:type: dataManagement
+:status: published
+:parent: Automatically Added Metacard Attributes
+:summary: How input tranformers add attributes to metacards.
+:order: 01
+
+=== {title}
+
+<<_available_input_transformers,Input Transformers>> create and populate metacards by parsing a resource.
+See <<_format_specific_attribute_mappings,Format Specific Attribute Mappings>> to see the attributes used for specific data formats.
+
+{branding} chooses which input transformer to use by:
+
+. Resolving the mimetype for the resource.
```

Comment:
@ricklarsen I have, of course, forgotten our house style. We do put periods at the end of bullet point items, right?

Reply:
Seems like something that should be added here: https://codice.atlassian.net/wiki/spaces/DDF/pages/6291516/Documentation+Style+Guide

Reply:
I scrolled through a couple sections of the documentation. Our bullet points end in periods.

Reply:
I added it to the style guide as well